project-metrics/metrics_service/metrics/presubmit_ignored_test.py

```diff
@@ -0,0 +1,72 @@
+"""Tests for presubmit-ignored metric."""
+
+import datetime
+import unittest
+
+from database import models
+from metrics import presubmit_ignored
+from test import metric_test_case
+
+
+class TestPresubmitIgnoredMetric(metric_test_case.MetricTestCase):
+
+  def tearDown(self):
+    super(TestPresubmitIgnoredMetric, self).tearDown()
+    session = self.Session()
+    session.query(models.Build).delete()
+    session.commit()
+
+  def new_metric_under_test(self):
+    return presubmit_ignored.PresubmitIgnoredMetric()
+
+  def testRecompute(self):
+    default_values = {
+        'started_at': datetime.datetime.now(),
+        'number': 1,
+        'duration': 1000,
+    }
+    session = self.Session()
+    session.add_all([
+        models.Build(state=models.TravisState.PASSED, **default_values),
+        models.Build(state=models.TravisState.CANCELLED, **default_values),
+        models.Build(state=models.TravisState.PASSED, **default_values),
+        models.Build(state=models.TravisState.ERRORED, **default_values),
+        models.Build(state=models.TravisState.FAILED, **default_values),
+    ])
+    session.commit()
+
+    self.metric.recompute()
+    self.assertLatestResultEquals(2)
```

Comment:
Explain in a one-line comment why it should be 2

Reply:
Done.